content/governance.md

```diff
@@ -34,19 +34,16 @@ New members may be proposed by any existing member by email to [prometheus-team]
 
 If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or deny their acceptance of team membership. This email will also be CC'd to [prometheus-team][team] for record-keeping purposes.
 
-If they choose to accept, the following steps are taken:
-
-* Team members are added to the [GitHub organization][gh] as _Owner_.
-* Team members are added to the [team mailing list][team].
-* Team members are added to the list of team members in this document.
-* New team members are announced on the [developers mailing list][devs] by an existing team member.
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
 
 Team members may retire at any time by emailing [the team][team].
 
-Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote, does not count towards the quorum, and may not make any changes to this and related documents any more. In case more than one removal vote runs at the same time, these limitations apply only to to respective removal votes, not globally.
```

Comment:
"to to respective removal votes" → "to each relevant removal vote individually"

This still doesn't clarify that the vote "I want to remove Alice, Bob, and Chris" is invalid.

Reply:
Rephrased completely to be more explicit.

Reply:
Happy?

Reply:
Beorn approved PR in full.